src/Epic/canRender.ts

```diff
@@ -0,0 +1,26 @@
+import { BrazeMessageProps } from './index.stories';
+
+/** These are in a seperate file to enable tree shaking of the logic deciding if a Braze message can be rendered
+ * this means the user won't download the Braze components bundle when the component can't be shown.
+ */
+export const canRenderEpic = (brazeMessageProps: BrazeMessageProps): boolean => {
```

Comment:
I wonder if it's worth migrating over the tests from DCR for this too, what do you think?

Reply:
I've added tests for this. I adapted the tests from DCR though they're slightly different. On DCR we're doing a mix of validation and translation of the props from the format we get from Braze to the format the Epic component expects in one step. It's (yet another) sign that borrowing the Epic component is a bit odd. I think this stuff would feel more natural if we brought the Epic component into this repo. That way we'd own the interface and wouldn't need to do this translation step from the Braze data.

Reply:
Agreed, but that'd be for another PR :)